src/SharedMath/Colors/RgbaColor.cs

```diff
@@ -17,5 +22,14 @@ public RgbaColor(byte r, byte g, byte b, byte a = 0xFF)
             this.B = b;
             this.A = a;
         }
+
+        public uint ToUint32() =>
+            ((uint)A << 24) | ((uint)R << 16) | ((uint)G << 8) | (uint)B;
```

Comment:
And I began to understand why they call ARGB. Can we change the sequence and add a parser from the hex format for the converter?

Reply:
I don't see why you've decided to use ARGB instead of RGBA, but whatever, let's change it if you think it's better this way. But why exactly?

Reply:
I can rename it back, for me it is not critical, on the contrary, I even like RGBA more. Just wpf / avalonia represents color as a string in ARGB order (for example, all approved files would have to be edited). Accordingly, we must be able to parse such an order that it will already slightly contradict the previous name of the class. Only on this seemed to me a good idea to do so. I no longer see the benefits of a _custom_ name